requirements.txt

```diff
@@ -8,7 +8,7 @@ wasabi>=0.4.0,<1.1.0
 catalogue>=0.2.0,<3.0.0
 ml_datasets==0.2.0a0
 # Third-party dependencies
-pydantic>=1.5.0,<1.7.0
+pydantic>=1.5.0,<1.8.0
```

Comment:
We want `>=1.7.0` here or python 3.9 will break.

Reply:
Okay, fair enough. (I guess technically we could use Python version-specific markers here but not sure if's worth the hassle 🤷‍♀️)

Reply:
Wait, I think we had a discussion about a bug fix on their end and it should be `>=1.7.1`.

Reply:
Yes, good point, I remember that. I'll update the pin accordingly.